Userland/Demos/CatDog/CatDog.cpp

```diff
@@ -100,6 +100,8 @@ bool CatDog::is_inspector() const
 
 void CatDog::timer_event(Core::TimerEvent&)
 {
+    using namespace AK::TimeLiterals;
```

Comment:
This is kinda ugly, especially because everyone needs to do it. Can we instead do this in TimeLiterals and add a normal `#ifdef USING_AK_GLOBALLY`?

Reply:
I'd rather not add these user defined literals to the global namespace until we get a better feel for how they're used, and if there's any other UDLs we start using. They seem somewhat nice so far.